care/facility/management/commands/load_medicines_data.py

```diff
@@ -24,6 +24,7 @@ def handle(self, *args, **options):
         MedibaseMedicine.objects.bulk_create(
```

Comment:
add a check to fallback to bulk create only if there are no medicines in the db to speed up the process